deployment/helm/skaha/skaha-config/launch-carta.yaml

```diff
@@ -39,18 +39,7 @@ spec:
       containers:
       - name: "${skaha.jobname}"
         env:
-        - name: skaha_hostname
-          value: "${skaha.hostname}"
-        - name: skaha_username
-          value: "${skaha.userid}"
-        - name: skaha_sessionid
-          value: "${skaha.sessionid}"
-        - name: HOME
-          value: "${SKAHA_TLD}/home/${skaha.userid}"
-        - name: PWD
-          value: "${SKAHA_TLD}/home/${skaha.userid}"
-        - name: OMP_NUM_THREADS
-          value: "${software.requests.cores}"
```

Comment:
Hi @at88mph - I'm not sure that moving these from the yaml files to the helm config is valuable, since they are specific to this very spot.  It kind of aligns with the point @rptaylor made in the multiple-mounts PR.  #681 .

Reply:
OK, done.  The `common` definition is still there as it relieves the necessity of changing every Job file each time.